dumb.svg

```diff
@@ -0,0 +1,32 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1024" height="560">
```

Comment:
We should probably standardize where all the "test" svgs go. They need the same path as mtgnewsbot.js I think.